main/ertp/guide/create-fungible-erights.md

```diff
@@ -0,0 +1,92 @@
+# Creating a fungible eright with ERTP
+
+## Definition of "fungible"
+
+A **fungible** asset means that different elements of this asset are interchangeable
+Currencies are interchangeable: if you have 10 one-dollar notes laying around and decide
+to take 5 of them, you do not care which 5 you will take. They're all "the same" as far 
+as counting value goes.
+
+This would not be the case for 10 house ownership rights each designating a different house
+House ownership right are not fungible, they are **specific**
+
+
+## Creating a fungible asset with ERTP
+
+### Creating and sending the asset
+
+In ERTP, digital assets are created by a [`mint`](./mint.html). Having access to the mint
+gives you the power to create more digital assets of the same type at
+will. For instance, let's say we want to create a new community
+currency called 'BaytownBucks'.
+You would first install the [ertp JavaScript package](https://www.npmjs.com/package/@agoric/ertp)
+(`npm install @agoric/ertp`) and then:
+
+```js
+import { makeMint } from '@agoric/ertp/core/mint';
+
+const baytownBucksMint = makeMint('BaytownBucks');
+```
+
+Great! Now let's use our mint to create 1000 new BaytownBucks.
+
+```js
+const purse = baytownBucksMint.mint(1000, 'community treasury');
```

Comment:
I see what you mean about the extent `1000` being used here rather than units. I'm torn on this since it would complicate the tutorial to explain units. Maybe let's try to use units anyways and see how complicated it makes it. 

If we use units, it would change to:

```
const baytownBucksAssay = baytownBucksMint.getAssay();
const bucks = baytownBucksAssay.makeUnits;
const bucks1000 = bucks(1000);
const purse = baytownBucksMint.mint(bucks1000, 'community treasury');
```
It's not great. I think we should explore ways of simplifying these steps.

Reply:
i removed the 3rd line, but kept your code

> It's not great. I think we should explore ways of simplifying these steps.

Agreed. Let's continue the conversation over at https://github.com/Agoric/ERTP/issues/238 and see where it lands. It'll certainly be easy to change the tutorial when there is an agreement and this agreement is implemented